package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@buildertrend/eslint-plugin-enterprise-extras",
   "description": "Extra eslint rules for enterprise environments focusing on React and Typescript",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Comment:
Normally new plugins require major version changes, but because this rule _requires_ configuration, I suppose it's good as is.

Reply:
![image](https://user-images.githubusercontent.com/9992362/218215114-0f1852fc-b73b-45e4-924a-41bb9e2cf789.png)

Reply:
Das wrong. Let me update lol